codecov.yml

```diff
@@ -1,9 +1,35 @@
 comment:
-  layout: 'reach, diff, flags'
+  layout: 'reach, diff, flags, files, header, footer, feedback'
+  behavior: default
+github_checks: false
 coverage:
   status:
     project: false
     patch: false
 flag_management:
   default_rules:
     carryforward: true
+component_management:
+  default_rules: # default rules that will be inherited by all components
+    statuses:
+      - type: project # in this case every component that doens't have a status defined will have a project type one
+        target: auto
+        informational: true
+        branches:
+          - '!main'
+  individual_components:
+    - component_id: api-graphql # this is an identifier that should not be changed
+      name: Portal GraphQL
+      paths:
+        - apps/api/**
+        - libs/api/domains/**
```

Comment:
Ah, so we can easily list which workspaces to combine in a single component? That's nice!

Can we have the same workspace in more than one component?

Reply:
Yes.